pyrobosim/pyrobosim/utils/pose.py

```diff
@@ -199,6 +199,34 @@ def __repr__(self):
         )
         return f"Pose: [{pos_str}, {quat_str}]"
 
+    def is_approx(self, other, rel_tol=1e-09, abs_tol=0.0):
+        if not (isinstance(other, Pose)):
+            raise TypeError("Expected a Pose")
+
+        a = np.array([self.x, self.y, self.z])
+        b = np.array([other.x, other.y, other.z])
+
+        # rel_tol = 1e-09 # setting rel_tol here makes it a float
+
+        # Just for debugging - rel_tol is a Pose for some reason?
+        print(
+            isinstance(a[0], float),
+            isinstance(a[1], float),
+            isinstance(a[2], float),
+            isinstance(b[0], float),
+            isinstance(b[1], float),
+            isinstance(b[2], float),
+            isinstance(rel_tol, float),
+            isinstance(abs_tol, float),
+        )
+
+        return np.allclose(a, b, rel_tol, abs_tol) and nearly_equivalent(
+            self.q, other.q
```

Comment:
Shouldn't the abs_tol and rel_tol also be passed into `nearly_equivalent`?

Reply:
I wasn't sure about this, transforms3d uses their own values